### ISSUE TITLE
[BUG] Crashing virt-handler due to the missing /virt_launcher.cil

### ISSUE THREAD

**Issue by avairavan** [harvester/harvester#805]
**Describe the bug**
I installed the harvester release version v0.2 on my kubernetes cluster. During the harvester installation, both the virt-handler pods crashed complaining about the missing /virt_launcher.cil file. virt-handler is using the container registry.suse.com/harvester-beta/virt-handler:0.40.0-1 which is missing the file.

**To Reproduce**
1. git clone the harvester git version v0.2
2. Install the harvester using the helm command - helm install harvester harvester --namespace harvester-system \
       --set longhorn.enabled=true,minio.persistence.storageClass=local-path,service.harvester.type=NodePort
3. List the pods using kubectl utility and observe that the virt-handler pod is on a crash loop. 

**Expected behavior**
All the pods in the harvester-system namespace should be in running state.

**Log**
{"component":"virt-handler","level":"info","msg":"Updating cluster config from KubeVirt to resource version '2925'","pos":"config-map.go:516","timestamp":"2021-04-29T16:28:24.890812Z"}
{"component":"virt-handler","level":"info","msg":"set verbosity to 2","pos":"virt-handler.go:422","timestamp":"2021-04-29T16:28:24.890894Z"}
{"component":"virt-handler","level":"info","msg":"SELinux is reported as 'permissive'","pos":"virt-handler.go:334","timestamp":"2021-04-29T16:28:24.904199Z"}
panic: failed to install virt-launcher selinux policy: failed to copy policy /var/run/kubevirt/virt_launcher.cil - err: failed to read a policy file /virt_launcher.cil: open /virt_launcher.cil: no such file or directory

**Environment:**
 - Harvester ISO version: v0.2.0-rc5
 - Installation Mode: app-mode
 - Underlying Infrastructure : Intel(R) Xeon(R) Silver 4210R CPU @ 2.40GHz

**Comment by gitlawr** [comment]
Can you show your Kubernetes/OS version?
Does it work if you disable SELinux?

**Comment by avairavan** [comment]
Kubernetes (K3s) version: v1.17.2+k3s1
CentOS version: 7.9.2009

**Comment by avairavan** [comment]
After disabling selinux, virt-handler is not crashing.

**Comment by gitlawr** [comment]
I can reproduce using the following steps:
1. Provision a CentOS 7 VM. Make sure SELinux is either Permissive or Enforcing.
2. Install the latest K3S.
3. helm install the harvester chart

result: virt-handler crashes.

After I edit the virt-handler daemonset, then change the handler image to `quay.io/kubevirt/virt-handler:v0.40.0`, and change the init container image to `quay.io/kubevirt/virt-launcher:v0.40.0`, the virt-handler pod turns to running status.

It could be an image packaging issue. The workaround is to update the image or disable SELinux.

**Comment by guangbochen** [comment]
@vasiliy-ul can u guys please take a look at how the image packaging was handled on SELinux with the SUSE image, thanks.

**Comment by vasiliy-ul** [comment]
Hi, thanks for reporting that. I managed to reproduce the issue on my test VM (fedora with selinux) and prepared the fix. Now rebuilding the containers...

Meanwhile, @guangbochen , what would be the best way to deliver the fix? I see two options:

- I can upload all the 5 Kubevirt images with the new tag `0.40.0-2`. Then the charts need to be updated.
- Alternatively I can try to overwrite the existing `virt-handler:0.40.0-1` image. In that case everyone needs to remove the old cached image e.g. `k3s ctr images rm registry.suse.com/harvester-beta/virt-handler:0.40.0-1` otherwise it will not be re-pulled.

**Comment by guangbochen** [comment]
The new tag with `0.40.0-2` would be preferred, thanks.

**Comment by vasiliy-ul** [comment]
@guangbochen , I've uploaded the images with the new tag `0.40.0-2`.

**Comment by futuretea** [comment]
I can confirm this has been fixed by tag 0.40.0-2

**Comment by ldevulder** [comment]
I installed Harvester version `master-5ea5fc31-head` with the same command as @avairavan on k3s and I can confirm that the issue is resolved:
```
# kubectl get pods -A | grep virt-handler
harvester-system   virt-handler-7j7c2                                     1/1     Running     0          3h25m
```

k3s was installed on an openSUSE MicroOS with SELinux set to Enforcing,
